runtime/runtime/tests/runtime_group_tools/mod.rs

```diff
@@ -205,10 +219,15 @@ impl RuntimeGroup {
                 });
                 state_records
                     .push(StateRecord::Contract { account_id, code: contract_code.to_vec() });
+                validators.push(AccountInfo {
+                    account_id: signer.account_id.clone(),
+                    public_key: signer.public_key.clone(),
+                    amount: if (i as u64) < num_existing_accounts { TESTING_INIT_STAKE } else { 0 },
```

Comment:
The check is already done above, so there is no need for this `if`:

```suggestion
                    amount: TESTING_INIT_STAKE,
```

Reply:
Done